### ISSUE TITLE
NBEATS: Zero inflated validation loss with shared Loss class.

### ISSUE THREAD

**Issue by kdgutier** [Nixtla/neuralforecast#65]
The reason why we used to have the numpy validation loss was to deal with the zeros in the mask. 
In the numpy loss zeros are not counted as zero they are just excluded, while the pytorch loss just masks the gradients by multiplying the entries.
